tools/amazon_agent_settings/agent_gemfile

```diff
@@ -0,0 +1,7 @@
+source "https://rubygems.org"
+gem "manageiq-gems-pending", ">0", :require => "manageiq-gems-pending", :git => "https://github.com/ManageIQ/manageiq-gems-pending.git", :branch => "master"
+gem "manageiq-smartstate", "~>0.1.5", :require => "manageiq-smartstate"
+gem "amazon_ssa_support", ">0", :require => "amazon_ssa_support", :git => "https://github.com/ManageIQ/amazon_ssa_support.git", :branch => "master"
+# Modified gems for gems-pending.  Setting sources here since they are git references
+gem "handsoap", "~>0.2.5", :require => false, :git => "https://github.com/ManageIQ/handsoap.git", :tag => "v0.2.5-5"
```

Comment:
Why do we need handsoap?  Our handsoap hacks were always just for VMware, and the AWS::SDK should't need it.

Reply:
I don't know this story, just saw manageiq-gem-pending depends on handsoap. Will remove it.